### ISSUE TITLE
Images not displaying in OJS production

### ISSUE THREAD

**Issue by kelynch** [pulibrary/openpublishing#56]
Images uploaded in OJS admin layer (logo/homepage) are not displaying in production environment http://openpublishing.princeton.edu/.

May be a recurrence of https://github.com/pulibrary/openpublishing/issues/39?

**Comment by bess** [comment]
I couldn't figure out how to replicate the issue, so I'm not sure whether this has been fixed. I did discover that the file permissions were different between staging and production, however, and that the ansible playbook that I would have expected to fix that issue was not fixing it. I updated the playbook and now the file permissions match staging. I've asked Yuan to confirm that the issue is fixed.

![Screen Shot 2022-11-03 at 1 37 36 PM](https://user-images.githubusercontent.com/65608/199798631-20ef0cd9-e7d4-4f23-979c-e3120d49d425.png)
![Screen Shot 2022-11-03 at 11 09 59 AM](https://user-images.githubusercontent.com/65608/199798635-51457281-936e-4dfd-b215-edc920215a1e.png)
![Screen Shot 2022-11-03 at 1 51 35 PM](https://user-images.githubusercontent.com/65608/199798636-04e55321-7591-407a-816f-396c8ca388df.png)
